thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js

```diff
@@ -66,12 +67,40 @@ export default Component.extend({
    */
   options: Object.keys(ANOMALY_OPTIONS_MAPPING),
 
+  /**
+   * Can be set by isWarning if data inconsistent
+   * @type {boolean}
+   */
+  warningValue: false,
+
   /**
    * A mapping of the status and a more human readable version
    * @type {Object}
    */
   optionsMapping: ANOMALY_OPTIONS_MAPPING,
 
+  /**
+   * Locally cache original time range on init
+   *
+   */
+  init() {
+    this._super(...arguments);
+    set(this, 'anomalyRangeOld', get(this, 'anomalyRange'));
+  },
+
+  /**
+   * Checks if anomalyRange has been changed since init
+   * @type {boolean}
+   */
+  isRangeChanged: computed(
+    'anomalyRange',
+    function () {
+      const anomalyRange = get(this, 'anomalyRange');
+      const anomalyRangeOld = get(this, 'anomalyRangeOld');
+      return (anomalyRange != anomalyRangeOld);
```

Comment:
This isn't gonna work, unfortunately. You'll need to compare the **start** and **end** of the anomaly entity with the anomaly range in the rca context

Reply:
more background - the user may change the investigation range and save a session. If the session is loaded later on, the anomalyRange will be whatever the user saved, rather than the anomaly's original value.

Reply:
@apucher thank you for the help - I updated the code as specified